### DISCUSSION TITLE
Duino-coin release for IOS

### DISCUSSION THREAD

**Discussion by filut123** [revoxhere/duino-coin#767] [Q&A]
Which file do I copy into Arduino Uno for Mac OS? I only see Windows and Linux. Please advise.
Malgorzata

**Comment by revoxhere** [comment]
Hello, the Arduino codes are the same for every operating system, so just upload which one you like to your board. However, to launch the softwares, you will need to to it [from source](https://github.com/revoxhere/duino-coin#manual-installation) as we don't have a MacOS release yet.

**Reply by filut123** [reply]
Thank you. Appreciate it.
